File: app/src/main/java/com/bresiu/codechallenge/model/Album.java
package com.bresiu.codechallenge.model;

public class Album {
	public long id;
	public long userId;
	public String title;

	public Album(long id, long userId, String title) {
		this.id = id;
		this.userId = userId;
		this.title = title;
	}
}

File: app/src/main/java/com/bresiu/codechallenge/db/entity/Album.kt
package com.bresiu.codechallenge.db.entity

import android.arch.persistence.room.Entity
import android.arch.persistence.room.ForeignKey
import android.arch.persistence.room.PrimaryKey

@Entity(foreignKeys = [(ForeignKey(entity = User::class, parentColumns = arrayOf("id"), childColumns = arrayOf("userId"), onDelete = ForeignKey.CASCADE))])
class Album {
    @PrimaryKey
    internal var id: Long = 0
    internal var userId: Long = 0
    internal var title: String? = null
}
